### PR TITLE
Update stdlib versions

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
$subject

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -79,7 +79,7 @@ ballerinaCommandVersion=1.3.15
 graphqlVersion=0.7.1-20230816-004300-12ef416
 
 # Protoc Tool
-protocToolVersion=0.1.2
+protocToolVersion=0.1.3-20230713-142600-0648ca1
```
Comment (dilanSachi) [inline gradle.properties:82]: shouldnt we bump this to a new minor version with java17 change?

Comment (MadhukaHarith92) [reply gradle.properties:82]: Yes, once @gabilang send the migration PR will bump to the next minor version.

Comment (kaneeldias) [reply gradle.properties:82]: Can we go ahead and merge this PR until protocTool is done?

Comment (MadhukaHarith92) [reply gradle.properties:82]: +1

**Review by malinthar — APPROVED** [review]

**Pull request merged**